roadmap.md

```diff
@@ -0,0 +1,84 @@
+# Nix is everywhere and invisible
+
+Software developers use Nix as a matter of course every day, mostly without even noticing.
+Nix runs trivially, anywhere.
```

Comment:
```suggestion
# Our vision for Nix: *the* utility behind all software development

Utilities, like water, electricity, or the internet are not like other products or inputs.
They are intentionally neutral, unopinionated, or even *boring*.
No one complains electricity or water is "misdesigned" or a the wrong tool for the job --- "a square peg in a round hole".
They also don't care what you use them for, and they are ubiquitous.
There are too many tasks that rely on them for anyone to keep track of.
When they do try to try to step outside their division of labor, like violating net neutrality, users don't like it.
Most of the time they just work, and during those times, you don't even think about them.
They just recede in the background.

All software development should be able to rely on Nix this same way.
Whatever you are trying to do, Nix should be an essential tool to support what you do, and yet one you don't even notice.
Whatever you opinions are on how packages should be written, dependencies specified, Nix should be able to help out.
Whatever machines you have at your disposal to build or deploy software, Nix should be able to work with them.

There should never be a reason not to use Nix.
```

Reply:
We decided to remove the "invisible" aspect.